src/styles/less/Hud.less

```diff
@@ -17,6 +18,11 @@
     flex-direction: column;
 }
 
+<<<<<<< HEAD
```

Comment:
Are these lines conflicts that still need to be resolved?

Reply:
resolved